src/main/java/com/trueffect/messages/Message.java

```diff
@@ -1,24 +1,39 @@
-    public static  String NOT_VALID_BIRTHDAY="The  birthday not is valid";
-    public static  String NOT_VALID_GENRE="The genre not is valid";           
-    public static  String NOT_VALID_FIRST_NAME="The first name size must be less than or equal to 50 characters";         
-    public static  String NOT_VALID_LAST_NAME="The last name size must be less than or equal to 50 characters";         
-    public static  String NOT_VALID_TYPE_IDENTIFIER="The type of dentifier not valid";     
-    public static  String NOT_VALID_IDENTIFIER="The dentifier not's  valid";     
-    public static  String DUPLICATE_IDENTIFIER="The identifier already exists";     
-    public static  String NOT_MEET_THE_AGE="To be registered must be greater than 15 years"; 
-    /*
-    * constant for resources
-    */
-    public static  String NOT_FOUND="The requested resource does not exist";
+
+    // Message of Empty
+    public static String EMPTY_TYPE_IDENTIFIER = "The type of identifier is required";
+    public static String EMPTY_IDENTIFIER = "The identifier is required";
+    public static String EMPTY_LAST_NAME = "The last name is required";
+    public static String EMPTY_FIRST_NAME = "The firt name is required";
+    public static String EMPTY_GENRE = "The genre is required";
+    public static String EMPTY_BITHDAY = "The birthday is required";
+    //Message dates incorrect
+    public static String NOT_VALID_TYPE_IDENTIFIER = "The type of identifier is not valid";
+    public static String NOT_VALID_IDENTIFIER = "The identifier is not valid";
+    public static String NOT_VALID_LAST_NAME = "The last name is not valid";
+    public static String NOT_VALID_FIRST_NAME = "The first name is not valid";
+    public static String NOT_VALID_GENRE = "The genre is not valid";
+    public static String NOT_VALID_BIRTHDAY = "The date format is not valid";
+    public static String NOT_SAME_TYPE = "The identifier is not the same type";
+    //Message de restrictions    
+    public static String SIZE_IDENTIFIER = "The identifier is very long, must be less than 10 characters";
+    public static String SIZE_LAST_NAME = "The last name is very long, must be less than 50 characters";
+    public static String SIZE_FIRST_NAME = "The first name is very long, must be less than 50 characters";
+    public static String DUPLICATE_IDENTIFIER = "The identifier already exists";
+    public static String NOT_MEET_THE_AGE = "To be registered must be greater than 15 years";
+    public static String THE_NAMES_ALREADY_EXIST = "The names already exist";
+    public static String NOT_VALID_AGE = "Must be  at least 15 years old";
+    // Message of results
+    public static String NOT_FOUND = "The requested resource does not exist";
+    public static String NOT_RESOURCE = "There is not resource";
+    // Message of permission
+    public static String NOT_HAVE_PERMISSION_TYPE_IDENTIFIER = "You do not have permission to modify type identifier";
```

Comment:
I think is better this :" The user don't have permission to modify type identifier" , let me know if you agree  or not with that